client/js/options.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
 const $ = require("jquery");
+const escapeRegExp = require("lodash/escapeRegExp");
```

Comment:
Before we merge, gotta test if this actually only brings the relevant function, or half of lodash with it.